pkg/utils/utils.go

```diff
@@ -415,3 +359,22 @@ func GetClusterNameFromKubectlCmd(cmd string) string {
 	}
 	return s
 }
+
+// ParseResourceArg parses the group/version/resource args and create a schema.GroupVersionResource
+func ParseResourceArg(arg string) schema.GroupVersionResource {
+	var gvr schema.GroupVersionResource
+	if strings.Count(arg, "/") >= 2 {
+		s := strings.SplitN(arg, "/", 3)
+		gvr = schema.GroupVersionResource{Group: s[0], Version: s[1], Resource: s[2]}
+	} else if strings.Count(arg, "/") == 1 {
+		s := strings.SplitN(arg, "/", 2)
+		gvr = schema.GroupVersionResource{Group: "", Version: s[0], Resource: s[1]}
+
+	}
+	return gvr
+}
+
+// TransformIntoTypedObject uses unstructured interface and creates a typed object
+func TransformIntoTypedObject(obj interface{}, typedObject interface{}) error {
+	return runtime.DefaultUnstructuredConverter.FromUnstructured(obj.(*unstructured.Unstructured).UnstructuredContent(), typedObject)
```

Comment:
This is nice.
Please set obj parameter type as `*unstructured.Unstructured` so that we don't have to typecast

Reply:
Done.

Reply:
For the review comment above this one, we can do that in the same block, but might not be such a pretty implementation as we can to typecast the object to get `involvedObject` field.